### PR TITLE
Various profile and community fixes

### PR DESCRIPTION
**Pull request by RiverRuby**
- [x]  title of dashboard internally is wrong
- [x]  make import button full length in dashboard
- [x]  move data to edit data once it’s been setup
- [x]  make profile pictures smaller
- [x]  remove interests from registration + profile
- [x]  move old leaderboard to another section
- [x]  other figma changes
- [x]  Login bugs
    - [x]  Login doesn’t exit if you mistype email
    - [x]  No loading on login
- [x]  leaderboard include current ring winners
- [x]  leaderboard is just top 5
- [x]  add mona to app

### CODE REVIEW

**Comment by stevenelleman** [comment]
Styling doesn't look right on profile: 

image

File: apps/frontend/src/pages/register.tsx [hunk 1]
```diff
@@ -232,30 +234,19 @@ const Register: React.FC = () => {
...
-      toast(SupportToast("", true, "Failed to create account. Please try again", ERROR_SUPPORT_CONTACT, errorToString(error)));
+      toast(
+        SupportToast(
+          "",
+          true,
+          "Failed to create account. Please try again",
+          ERROR_SUPPORT_CONTACT,
+          errorToString(error)
+        )
+      );
```
Comment (stevenelleman) [inline apps/frontend/src/pages/register.tsx:245]: I'll be sure to follow this pattern in the future.

File: apps/frontend/src/pages/community/index.tsx [hunk 2]
```diff
@@ -533,10 +561,43 @@ export default function CommunityPage() {
...
             <div className="flex flex-col gap-2">
               <span className="text-base font-bold text-primary font-sans">
-                Coming soon!
+                Coming soon
               </span>
               <ComingSoonCommunityGoals />
             </div>
+            <div className="flex flex-col gap-2">
+              <span className="text-base font-bold text-primary font-sans">
+                {`Past dashboards`}
```
Comment (stevenelleman) [inline apps/frontend/src/pages/community/index.tsx:570]: I like this feature, I'm a sucker for digging into historical patterns / trends.

**Review by stevenelleman — APPROVED** [review]

**Comment by stevenelleman** [comment]
@RiverRuby UI feature request -- could we add "click to refresh" next to the import button for folks who have already imported? This would only be necessary until automatic refresh logic is added.

File: apps/frontend/src/pages/register.tsx [hunk 3]
```diff
@@ -352,11 +343,6 @@ const Register: React.FC = () => {
           />
         )}
         {displayState === DisplayState.CREATING_ACCOUNT && <CreatingAccount />}
-        {displayState === DisplayState.LANNA_DISCOVER_CONNECTIONS && (
-          <LannaDiscoverConnections
-            onSubmit={handleLannaDiscoverConnectionsSubmit}
-          />
-        )}
```
[the remark this answers is not in the document]
Comment (RiverRuby) [reply apps/frontend/src/pages/register.tsx:359]: It Is No More

**Pull request merged**